djangosaml2/views.py

```diff
@@ -123,7 +125,9 @@ def login(request,
     # SAML_IGNORE_AUTHENTICATED_USERS_ON_LOGIN setting. If that setting
     # is True (default value) we will redirect him to the came_from view.
     # Otherwise, we will show an (configurable) authorization error.
-    if callable_bool(request.user.is_authenticated):
+    mfa_account = request.session.get("mfa_done_accounts", None)
```

Comment:
is this related to any particular app or external dependecy?

do you use/see this? https://github.com/xi/django-mfa3